src/components/companion/CompaniesList.tsx

```diff
@@ -0,0 +1,258 @@
+"use client";
+
+import { motion } from "framer-motion";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { TopNav } from "./navigation/top-nav";
+import { SideNav } from "./navigation/side-nav";
```

Comment:
small note that Izzy has refactored the Nav bars to go under a single component, so you may need to fix a merge conflict